runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsOverlay.java

```diff
@@ -70,7 +70,7 @@
 	// The 15 pixel gap between each drawn ground item.
 	private static final int STRING_GAP = 15;
 	// Threshold for highlighting items as blue.
-	private static final int LOW_VALUE = 20_000;
+	static final int LOW_VALUE = 20_000;
```

Comment:
Is there a reason why these are no longer private?

Reply:
Yes those are used in the GroundItemsPlugin class